packages/react-instantsearch-hooks/src/__tests__/usePoweredBy.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { usePoweredBy } from '../usePoweredBy';
+
+describe('usePoweredBy', () => {
+  test('returns the connector render state', () => {
+    const { url } = usePoweredBy();
+
+    expect(url).toBe(
+      `https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_content=${
+        window.location?.hostname || ''
+      }&utm_campaign=poweredby`
```

Comment:
Can we inline the value in the test instead of reusing the implementation code? We can't guess if the expectation is correct.

Reply:
I did this because I wasn't sure it would always be the same in CI. But I guess we can try.

Reply:
Addressed in 3f06720. Also added a test for when `window.location` isn't available in 1c93b38.